linux_os/guide/services/base/service_cgred_disabled/rule.yml

```diff
@@ -13,14 +13,15 @@ rationale: |-
     Unless control groups are used to manage system resources, running the cgred service
     service is not necessary.
 
-severity: unknown
+severity: low
 
 identifiers:
     cce@rhel6: 27252-6
     cce@rhel7: 80255-3
+    cce@rhel8: 82398-9
```

Comment:
@shawndwells we should return this CCE back to the pool. RHEL8 doesn't appear to have `cgred` service anymore.

Reply:
Done. Also removed rhel8 from the prodtype.